qemu/tests/cfg/blockdev_commit_to_nospace.cfg

```diff
@@ -0,0 +1,49 @@
+- blockdev_commit_to_nospace:
+    type = blockdev_commit_to_nospace
+    virt_test_type = qemu
+    only Linux
+    device_tag = 'data'
+    images += " ${device_tag}"
+    force_create_image_data = yes
+    remove_image_data = yes
+    start_vm = yes
+    kill_vm = yes
+    storage_pools = default
+    storage_type_default = "directory"
+    storage_pool = default
+    image_size_data = 2G
+    snapshot_tags_data = sn1
+
```

Comment:
we may also add image_format_data = qcow2, for we cannot create 2G data image in other format in a smaller space

Reply:
Have updated, thanks zhenchao.